### PR TITLE
vrops-exporter: nvme datastore utilization alert

### PR DESCRIPTION
**Pull request by christopherhans**
New (local) NVMe swap datastore. According to Thomas Thimm no critical alert needed. As requested, created separate alert with 85% threshold.

### CODE REVIEW

File: prometheus-exporters/vrops-exporter/alerts/datastore.alerts [hunk 1]
```diff
@@ -173,6 +173,22 @@ groups:
       description: "{{ $labels.type }} datastore {{ $labels.datastore }} utilization > 90%. ({{ $labels.vcenter }}, {{ $labels.datacenter }})"
       summary: "{{ $labels.type }} datastore {{ $labels.datastore }} utilization > 90%. ({{ $labels.vcenter }}, {{ $labels.datacenter }})"
 
+  - alert: NVMeDataStoreCapacityWarning
+    expr: >
+      max_over_time(vrops_datastore_diskspace_total_usage_gigabytes{datastore=~".+swap"}[10m]) / max_over_time(vrops_datastore_diskspace_capacity_gigabytes[10m]) >=  0.85
```
Comment (viennaa) [inline prometheus-exporters/vrops-exporter/alerts/datastore.alerts:178]: `max_over_time` should not be needed anymore, should it?

Comment (christopherhans) [reply prometheus-exporters/vrops-exporter/alerts/datastore.alerts:178]: The only gab I found today was because of a hickup in a single collector.

Comment (viennaa) [reply prometheus-exporters/vrops-exporter/alerts/datastore.alerts:178]: remove it completely then

**Comment by christopherhans** [comment]
Removed max_over_time for all alerts and adjusted context.

File: prometheus-exporters/vrops-exporter/alerts/datastore.alerts [hunk 2]
```diff
@@ -47,29 +47,29 @@ groups:
 
   - alert: EphemeralDataStoreCapacityWarning
     expr: >
-      max_over_time(vrops_datastore_diskspace_total_usage_gigabytes{type=~"ephemeral"}[10m]) / max_over_time(vrops_datastore_diskspace_capacity_gigabytes[10m]) >= 0.8
+      vrops_datastore_diskspace_total_usage_gigabytes{type=~"ephemeral"}/ vrops_datastore_diskspace_capacity_gigabytes >= 0.8
```
Comment (viennaa) [inline prometheus-exporters/vrops-exporter/alerts/datastore.alerts:50]:
```suggestion
      vrops_datastore_diskspace_total_usage_gigabytes{type=~"ephemeral"} / vrops_datastore_diskspace_capacity_gigabytes >= 0.8
```

**Review by viennaa — APPROVED** [review]

**Pull request merged**